src/playercards/PlayerCardSpawner.ttslua

```diff
@@ -105,8 +105,16 @@ Spawner.spawn = function(cardList, pos, rot, callback)
     scaleY = 1,
     scaleZ = cardList[1].data.Transform.scaleZ,
   }
+  local sidewaysDeck = true
   for _, spawnCard in ipairs(cardList) do
     Spawner.addCardToDeck(deck, spawnCard.data)
+    if not spawnCard.data.SidewaysCard then
```

Comment:
If you want to get fancy here, the if can be replaced with one line:
sidewaysDeck = sidewaysDeck and spawnCard.data.SidewaysCard

(Hopefully got the Lua syntax right there)